### ISSUE TITLE
Site randomly fails to load

### ISSUE THREAD

**Issue by coinop-logan** [olekon/justsmartcontracts#2]
I'm not sure what the root of the issue is, but going to justsmartcontracts.dev will randomly deliver simply a blank screen.

In at least one of these cases, the console eventually spat out `SyntaxError: "" literal not terminated before end of script`

**Comment by olekon** [comment]
Do you have any smart contracts in the list?
if yes, attach please the contents of the browser localstorage

**Comment by coinop-logan** [comment]
[localStorage-contracts.txt](https://github.com/olekon/justsmartcontracts/files/4254271/localStorage-contracts.txt)

**Comment by olekon** [comment]
I have no problems with this list. Please provide complete localstorage content. How often does it happen? What browser do you use?

**Comment by coinop-logan** [comment]
I'm using Chrome and Firefox, and both occasionally do this. It seems to happen on Firefox more often.

I'm not sure of a way to directly download localStorage content from Firefox, so I've copied each item (there are three) to the attached file. The first two lines are the first two items; then I've included the long contracts item (same as before) for convenience.

Happy to provide any more info! Thanks very much for these quick responses. This tool is extremely helpful in my day-to-day work. I'm also happy to communicate more directly if you like; feel free to email me at coinop.logan@gmail.com and we can perhaps jump on a chat.
[localStorage-justsmartcontracts.txt](https://github.com/olekon/justsmartcontracts/files/4260137/localStorage-justsmartcontracts.txt)
